src/themes/default/components/core/blocks/Microcart/Microcart.vue

```diff
@@ -49,7 +49,7 @@
       {{ $t('to find something beautiful for You!') }}
     </div>
     <ul v-if="productsInCart.length" class="bg-cl-primary m0 px40 pb40 products">
-      <product v-for="product in productsInCart" :key="product.checksum || product.sku" :product="product" />
+      <product v-for="product in productsInCart" :key="product.checksum || `${product.sku}-${getRandomNumber()}`" :product="product" />
```

Comment:
Isn’t checksum doing the job here? The options are set in the item.product_option so two products with different options should always have different checksum aren’t they?